### ISSUE TITLE
`Tag` down-arrow too small

### ISSUE THREAD

**Issue by ovari** [moneymanagerex/moneymanagerex#6280]
1. `Tags` down-arrow too small and not consistent with the other down-arrows.

![Transaction Filter](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/4e561306-efda-44a6-957c-8d2a19d5dd2e)

![New Transaction](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/f2170de6-4149-476c-9bef-afeb48e96fe8)

2. `Date` down-arrows (i.e. `˅`) have different down-arrows than other fields down-arrows (i.e. `▼`).

Thank you

cc: @n-stein

```
Money Manager Ex
• Version: 1.7.0-Beta.1 64-bit
• Built on: Nov 3 2023 03:16:23
• Database version: 19 (aes128cbc)
• Git commit: 0898c7b42 (2023-11-02)
• Git branch: master
MMEX is using the following support products:
• wxWidgets 3.2.3 (wxGTK 3.24)
• wxSQLite3 4.9.1 (SQLite 3.40.0)
• RapidJSON 1.1.0
• Lua 5.3.5
• libcurl/8.4.0-DEV OpenSSL/3.1.4 zlib/1.3 nghttp2/1.56.0
• gettext 0.22
• apexcharts.js
Built with:
• CMake 3.27.7
• GNU Make 4.4.1
• GCC 13.2.0
Running on:
• GNOME.45 (Flatpak runtime)
• GNOME 45 (Flatpak runtime) ""
• Linux 5.15.0-88-generic x86_64
• X-Cinnamon
• System Locale: en_AU.UTF-8
• User Interface Language: en_US.UTF-8
• 1440x810 24-bit 96x96ppi
```

**Comment by ovari** [comment]
3. There is no blue around the field when the `Tags` field has focus.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/605dcbdf-5397-4e0b-84c4-85b8889af2e5)

**Comment by n-stein** [comment]
@ovari please test that change, it looks better on my test box:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/0c3ec993-6fbf-49bb-affa-dc06b36e23cf)

**Comment by ovari** [comment]
@n-stein the `Payee` drop-down-arrow on your image seems to be a lighter than the other drop-down-arrows. It might be `Not a bug` as there are no `Payee`s?

**Comment by ovari** [comment]
> please test that change,

Thank you, can test once merged.

**Comment by n-stein** [comment]
> the `Payee` drop-down-arrow on your image seems to be a lighter than the other drop-down-arrows. It might be `Not a bug` as there are no `Payee`s?

Correct, that was a new DB with no Payees created, so the drop-down was disabled.

**Comment by ovari** [comment]
@n-stein the down-arrow looks better; however, when clicking on the down-arrow the drop-down box is not aligned. Should be positioned more to the left and lower down in both `New Transaction` and `Transaction Filter` dialogs.

![New Transaction](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/e20a3595-b0d1-415f-a0c0-774e5594d370)

![Transaction Filter](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/901f00eb-885b-4b2f-bf30-7a1daa9ad2bf)

Thank you

```
Money Manager Ex
• Version: 1.7.0-Beta.1 64-bit
• Built on: Nov 22 2023 23:14:12
• Database version: 19 (aes128cbc)
• Git commit: 072a65bbd (2023-11-22)
• Git branch: master
MMEX is using the following support products:
• wxWidgets 3.2.3 (wxGTK 3.24)
• wxSQLite3 4.9.1 (SQLite 3.40.0)
• RapidJSON 1.1.0
• Lua 5.3.5
• libcurl/8.4.0-DEV OpenSSL/3.1.4 zlib/1.3 nghttp2/1.56.0
• gettext 0.22
• apexcharts.js
Built with:
• CMake 3.27.7
• GNU Make 4.4.1
• GCC 13.2.0
Running on:
• GNOME.45 (Flatpak runtime)
• GNOME 45 (Flatpak runtime) ""
• Linux 5.15.0-89-generic x86_64
• X-Cinnamon (X11)
• System Locale: en_AU.UTF-8
• User Interface Language: en_US.UTF-8
• 1440x810 24-bit 96x96ppi
```

**Comment by whalley** [comment]
MacOS:
![CleanShot 2023-11-28 at 20 09 37](https://github.com/moneymanagerex/moneymanagerex/assets/487345/cc05155d-dbff-46b6-9fbe-7f531fc7a0a7)

**Comment by n-stein** [comment]
I think everything looks ok now.

Windows:
![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/a6624109-55b0-4231-9d4c-cc9ac7d4cb89)

Linux:
![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/936137f0-19f0-46af-a469-7c1dbba3a220)

macOS (note - no dropdown):
![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/c258ec40-1c62-4609-bbed-d59d95f68e12)

**Comment by whalley** [comment]
Yes, looks good on MacOS.
